systers_portal/static/css/style.css

```diff
@@ -771,3 +771,11 @@ input[id=go-btn]{
 height:30px;
 font-family: 'Cairo';
 }
+li[id=nav-menu]:hover{
+background-color:#54bceb;
+}
+@media only screen and (min-width: 768px) {
+  .dropdown:hover .dropdown-menu {
+    display: block;
+  }
+}
```

Comment:
I rendered the search view on Mobile

[image](https://user-images.githubusercontent.com/8980971/86935716-c4ba9100-c15a-11ea-9214-90a8ac854c7c.png)

Also, when you hover over a menu, the colors change to something unreadable

Reply:
Also on this point, I'd like to add the fact that it was supposed to look like [this](https://anitab.org/) and when you hover the black color should actually change to white and not just the change in the background color.

Reply:
@ritwickraj78 Did you look into this?

Reply:
Exactly 😬

Reply:
![Screenshot from 2020-07-09 12-23-34](https://user-images.githubusercontent.com/43514828/87007078-4a802000-c1df-11ea-8480-4ab36d26f0c7.png)
@sakshi1499 Could you elaborate a bit more? Didnt get you exactly

Reply:
@ritwickraj78 The website that I mentioned is anitaB's official website. Actually, when we hover the blue background on the navbar items has black colored text right? I was saying like the websites of anitaB, make that text white. So add the property of color: white;